README.md

```diff
@@ -4,11 +4,17 @@ Common loaders for Music Information Retrieval (MIR) datasets. Find the API docu
 ![CI status](https://github.com/mir-dataset-loaders/mirdata/actions/workflows/ci.yml/badge.svg)
 ![Formatting status](https://github.com/mir-dataset-loaders/mirdata/actions/workflows/formatting.yml/badge.svg)
 ![Linting status](https://github.com/mir-dataset-loaders/mirdata/actions/workflows/lint-python.yml/badge.svg)
```

Comment:
So, are we keeping these badges in the end?

Reply:
https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge#using-the-branch-parameter would this fix it?

Reply:
No, unless we create tests that run on _master_.